### PR TITLE
Feature/cache queries and add combined indicators

### PR DESCRIPTION
**Pull request by bimal125**
- Addresses https://github.com/collective-service/ifrc-gates-backend/issues/137

## Changes

* Added separate query for combined indicators
* Cached overview table data
* Cached overview map data
* Cached combined indicators

Mention related users here if any.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)

### CODE REVIEW

File: apps/visualization/utils.py [hunk 1]
```diff
@@ -361,3 +370,101 @@ def format_indicator_value(iso3, qs_map):
...
+
+    data = []
+    for thematic in thematics:
+        data.append(
+            {
+                'thematic': thematic,
+                'topics': [
+                    {
+                        'topic_name': topic['topic'],
+                        'topic_description': topic['topic_description'],
```
Comment (Rup-Narayan-Rajbanshi) [inline apps/visualization/utils.py:440]: I think It seems to raise error "topic is not defined",

**Review by Rup-Narayan-Rajbanshi — APPROVED** [review]

File: apps/visualization/dataloaders.py [hunk 2]
```diff
@@ -32,5 +33,14 @@ def indicator_value_regional_load(keys: List[int]):
     return [_map[key] for key in indicator_id_list]
 
 
+def population_size_load(keys: List[str]):
+    qs = CountryProfile.objects.filter(iso3__in=keys).values('iso3', 'population_size')
+    _map = defaultdict(str)
```
Comment (thenav56) [inline apps/visualization/dataloaders.py]: Should be int right?

File: apps/visualization/filters.py [hunk 3]
```diff
@@ -301,3 +301,27 @@ def filter_indicator_id(self, queryset):
         ).first().topic
         print(topic)
```
Comment (thenav56) [inline apps/visualization/filters.py]:
```suggestion
```

File: apps/visualization/utils.py [hunk 4]
```diff
@@ -361,3 +375,172 @@ def format_indicator_value(iso3, qs_map):
...
+        elif type == REGIONAL_LEVEL:
+            return {
+                'indicator_value': Max('indicator_value_regional'),
+            }
+        elif type == GLOBAL_LEVEL:
+            return {
+                'indicator_value': Max('indicator_value_global'),
+            }
+        else:
+            return {}
```
Comment (thenav56) [inline apps/visualization/utils.py]:
```suggestion
        return {}
```

**Review by thenav56 — CHANGES_REQUESTED** [review]
Looks good. Few changes.

**Pull request merged**